homeassistant/components/glances/const.py

```diff
@@ -40,164 +44,183 @@ class GlancesSensorEntityDescription(SensorEntityDescription):
         name_suffix="free",
         native_unit_of_measurement=DATA_GIBIBYTES,
         icon="mdi:harddisk",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="memory_use_percent",
         type="mem",
         name_suffix="RAM used percent",
         native_unit_of_measurement=PERCENTAGE,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="memory_use",
         type="mem",
         name_suffix="RAM used",
         native_unit_of_measurement=DATA_MEBIBYTES,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="memory_free",
         type="mem",
         name_suffix="RAM free",
         native_unit_of_measurement=DATA_MEBIBYTES,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="swap_use_percent",
         type="memswap",
         name_suffix="Swap used percent",
         native_unit_of_measurement=PERCENTAGE,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="swap_use",
         type="memswap",
         name_suffix="Swap used",
         native_unit_of_measurement=DATA_GIBIBYTES,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="swap_free",
         type="memswap",
         name_suffix="Swap free",
         native_unit_of_measurement=DATA_GIBIBYTES,
         icon="mdi:memory",
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="processor_load",
         type="load",
         name_suffix="CPU load",
-        native_unit_of_measurement="15 min",
         icon=CPU_ICON,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="process_running",
         type="processcount",
         name_suffix="Running",
-        native_unit_of_measurement="Count",
         icon=CPU_ICON,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="process_total",
         type="processcount",
         name_suffix="Total",
-        native_unit_of_measurement="Count",
         icon=CPU_ICON,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="process_thread",
         type="processcount",
         name_suffix="Thread",
-        native_unit_of_measurement="Count",
         icon=CPU_ICON,
+        state_class=SensorStateClass.MEASUREMENT,
     ),
     GlancesSensorEntityDescription(
         key="process_sleeping",
         type="processcount",
         name_suffix="Sleeping",
-        native_unit_of_measurement="Count",
+        native_unit_of_measurement="",
```

Comment:
We should not use emtpy string as unit of measurement. Please remove this line so we default to `None`.

Reply:
Oh missed that, will address it

Reply:
<https://github.com/home-assistant/core/pull/73983>